sdk/resourcemanager/azure-resourcemanager-containerinstance/src/main/java/com/azure/resourcemanager/containerinstance/models/ContainerGroup.java

```diff
@@ -1180,6 +1183,14 @@ interface WithDnsPrefix {
              * @return the next stage of the definition
              */
             WithCreate withDnsPrefix(String dnsPrefix);
+
+            /**
+             * Specifies the reuse policy of the DNS name label be to the object's domain name label can be reused in specified scope.
+             *
+             * @param reusePolicy the DNS name label reuse policy
+             * @return the next stage of the definition
+             */
+            WithCreate withAutoGeneratedDomainNameLabelScope(DnsNameLabelReusePolicy reusePolicy);
```

Comment:
Can this property be updated?

Reply:
The `ContainerGroup` interface does not contain any Update step, so I think `ContainerGroup` is a non-updatable service.
![image](https://github.com/user-attachments/assets/280c7267-5883-4599-b4f3-6cb02db36e68)